deploy/scripts/src/common/utils/custom_metric/counter.ts

```diff
@@ -0,0 +1,4 @@
+import { Counter } from 'k6/metrics'
+
+export const iterationsStarted = new Counter('Iterations_Started')
+export const iterationsCompleted = new Counter('Iterations_Completed')
```

Comment:
```suggestion
export const iterationsStarted = new Counter('iterations_started')
export const iterationsCompleted = new Counter('iterations_completed')
```
Suggest making them lowercase in line with other k6 metrics, but otherwise okay

Reply:
Done, thank you